src/Models/ResponseModels/MaterialResponse.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SolutionsService.Models.ResponseModels
+{
+    public class MaterialResponse
```

Comment:
Does this actually need to be a separate ResponseModel if we remove the bidirectional relationship in the database model? Or do we want to enforce strict segregation between database and response models? If so, why?

Reply:
We should not remove the bidirectional relationship in the database model + validation on request/response level should not mix with database validation.